konvoy: Update docs for konvoy release v1.5.0-rc.1 (#3055)
mesosphere/dcos-docs-site: pages/ksphere/konvoy/1.5/tutorials/disable-audit-logs/index.md

---
layout: layout.pug
navigationTitle: Disable Audit Log Collection
title: Disable Audit Log Collection
menuWeight: 11

> excerpt: Audit Logs are collected via the FluentBit Addon and enabled for collection by default. In some cases this may not be necessary and turning off audit-log-collection would be ideal. Here we show the procedure to disable Audit Log Collection.

Reviewer: ```suggestion
excerpt: Audit Logs are enabled and collected by default. If needed, you can disable Audit Log Collection.
```

enterprise: false
---

### Disable Audit Log Collection

> Audit Logs are collected via the FluentBit Addon and enabled for collection by default. In some cases this may not be necessary and turning off audit-log-collection would be ideal. Here we show the procedure to disable Audit Log Collection.

Reviewer: ```suggestion
Audit Logs are collected using the FluentBit Addon and enabled for collection by default. In some cases this may not be necessary and turning off audit-log-collection would be ideal. This procedure shows how to disable Audit Log Collection.
```

#### Before you begin

> These procedures require that you have the FluentBit addon enabled. If this is not the case then no further action is required.

Reviewer: ```suggestion
Thos procedure requires that you have the FluentBit addon enabled. If this is not the case then no further action is required.
```

#### Disable Audit Log Collection

1.  Update your `cluster.yaml` file’s `ClusterConfiguration` section with the following `values` entries to the `fluentbit` addon entry.

    ```yaml
    ---
    kind: ClusterConfiguration
    apiVersion: konvoy.mesosphere.io/v1beta1
    metadata:
    name: my-cluster
    creationTimestamp: "2020-06-05T16:57:18Z"
    spec:
    kubernetes:
        version: 1.16.8
    …
    addons:
    - configRepository: https://github.com/mesosphere/kubernetes-base-addons
        …
        addonsList:
        …
        - name: fluentbit
          enabled: true
          values: |
            audit:
              enabled: false
        …
    ```

>     (Optional) Also, if you wish to remove the Audit Log Dashboard from Kibana, you may update the `cluster.yaml` file with the following `values` entry of kibana

Reviewer: ```suggestion
    (Optional) Also, if you wish to remove the Audit Log Dashboard from Kibana, you can update the `cluster.yaml` file with the following `values` entry of kibana
```
